src/main/java/com/acciente/oacc/DomainPermission.java

```diff
@@ -21,8 +21,8 @@
  * The interface for the type of permission that governs domain management.
  * <p/>
  * A DomainPermission is the type of permission used to grant an accessor permission to manage
- * a domain. A DomainPermission is never application-defined, bu is always a system permission;
- * the supported system permissions values are *SUPER-USER and *CREATE-CHILD-DOMAIN.
+ * a domain. A DomainPermission is never application-defined, but is always a system permission;
+ * the supported system permissions values are *SUPER-USER, *CREATE-CHILD-DOMAIN and *DELETE.
```

Comment:
Good catch, Seb - thanks for pointing this out.
I've updated the javadoc comment accordingly and credited you in 51bffbe